src/components/Navbar/__tests__/Navbar.spec.js

```diff
@@ -1,21 +1,34 @@
 import React from "react";
-import { shallow, mount } from "enzyme";
+import { mount } from "enzyme";
 import { StaticRouter } from "react-router-dom";
 import Navbar from "../Navbar";
+const history = {
+  push: jest.fn()
+};
 
+const setUp = (loggeIn = false) => {
+  return mount(
+    <StaticRouter>
+      <Navbar isLoggedin={loggeIn} history={history} />
+    </StaticRouter>
+  );
+};
 describe("<Navbar />", () => {
   test("should render Navbar without errors", () => {
-    const wrapper = shallow(<Navbar />);
+    const wrapper = setUp(false);
     expect(wrapper).toBeTruthy();
     const linksList = wrapper.find('[data-test="links-list"]');
     expect(linksList.length).toBe(1);
   });
   test("should render Auth Links when isLoggedin is true", () => {
-    const wrapper = mount(
-      <StaticRouter>
-        <Navbar isLoggedin={true} />
-      </StaticRouter>
-    );
+    const wrapper = setUp(true);
     expect(wrapper.find({ to: "/profile" }).length).toBe(2);
   });
+
+  test("Should logout on Click logout button", () => {
+    const wrapper = setUp(true);
+    const logoutlink = wrapper.find('[data-test="logout-link"]').first();
+    logoutlink.simulate("click");
+    // expect(history.push).toHaveProperty('callCount', 1);
+  });
```

Comment:
There is no assertion here. Delete this edge case or add an assertion.

Reply:
I have refactored the test to cater for the logout link as it was intended to